db/migrate/20150213005103_resequence_broken_sequences.rb

```diff
@@ -0,0 +1,27 @@
+class ResequenceBrokenSequences < ActiveRecord::Migration
+  def create_progress_bar(total)
+    ProgressBar.create(format: "(\e[32m%c/%C\e[0m) %a |%B| \e[31m%e\e[0m ",
+                       progress_mark: "\e[32m/\e[0m",
+                       total: total)
+  end
+
+  def change
```

Comment:
@gdpelican would be good to continue walking you through this, especially these migrations to check that I'm not going to ruin anything. They seem to work well on a production clone though.